src/Az.Test.props

```diff
@@ -25,6 +25,11 @@
     <DefineConstants>TRACE;RELEASE;NETSTANDARD</DefineConstants>
   </PropertyGroup>
 
+  <ItemGroup>
+    <ProjectReference Condition="'$(IncludeCommonTestProject)' != 'false'" Include="$(MSBuildThisFileDirectory)\..\tools\ScenarioTest.ResourceManager\ScenarioTest.ResourceManager.csproj" />
```

Comment:
Move the `Condition="'$(IncludeCommonTestProject)' != 'false'"` to the ItemGroup and remove it from each ProjectReference. Then, change the paths to `Include="$(RepoTools)ScenarioTest.ResourceManager\ScenarioTest.ResourceManager.csproj"` for each.